Patch taken from the Desktop consolidation.
See BugDB CR #15471092
SUNBT6685564-CERES E6552323 fails with optimizations >xO1: Assert has been viola
for more details.

This patch will not be sent upstream.

diff -up poppler-0.10.6/poppler/Makefile.am.orig poppler-0.10.6/poppler/Makefile.am
--- poppler-0.10.6/poppler/Makefile.am.orig	2009-03-29 22:50:51.000000000 +0100
+++ poppler-0.10.6/poppler/Makefile.am	2009-04-20 13:08:57.621075691 +0100
@@ -204,6 +204,8 @@ poppler_include_HEADERS =	\
 
 endif
 
+JBIG2Stream.lo: CXXFLAGS += -xO1 
+
 libpoppler_la_SOURCES =		\
 	$(splash_sources)	\
 	$(cairo_sources)	\


